lib/interceptor.js

```diff
@@ -8,11 +8,24 @@ var interceptor = {
     }
     done(window[NAMESPACE]);
   },
+
+  excludeUrls: function excludeUrls(urls, done) {
+    var NAMESPACE = '__webdriverajax';
+    if (window[NAMESPACE] != undefined) {
+      Array.prototype.push.apply(window[NAMESPACE].excludedUrls, urls);
```

Comment:
My original suggestion was to do something like this:
```suggestion
  excludeUrls: function excludeUrls(urls, done) {
    var NAMESPACE = '__webdriverajax';
    if (window[NAMESPACE] != undefined) {
      // Convert the strings to regular expressions once.
      var exprs = urls.map(function (obj) {
        return new RegExp(obj.source, obj.flags);
      });
      Array.prototype.push.apply(window[NAMESPACE].excludedUrls, exprs);
```
In other words, when you receive the data from the node process, convert it once into a regexp.

(The above code assumes the node process has converted all user input into objects with two properties, "source" and "flags." For user-provided strings, "flags" should be `undefined`.)